src/docbkx/agents/jvm.xml

```diff
@@ -114,12 +114,37 @@ java -javaagent:agent.jar=port=7777,host=localhost]]></programlisting>
         <td><constant>host</constant></td>
         <td>
           Hostaddress to which the HTTP server should bind to. If "*" or "0.0.0.0" is
-          given, the servers binds to every network interface.
+          given, the servers binds to every network interface. This setting is mutually exclusive to <em>ipmatch</em> and <em>nicmatch</em>. 
+          Do not combine this settings. Bad configuration or errors resolving network informations will fallback to loopback interface for security reasons.
         </td>
         <td>
           <constant>localhost</constant>
         </td>
         </tr>
+        <tr>
+        <td><constant>ipmatch</constant></td>
```

Comment:
I wonder whether we could avoid to a new configuration key like `nicmatch` or `ipmatch` and fold everything into the existing `host` variable. This approach would have multiple advantages:

* Multiple config-clashes would be avoided automatically
* Don't blow up the already huge configuration surface

So my suggestion would be to introduce multiples prefix like `nic:` or `ip:` which then trigger the different modes. Maybe even better just a `match:` if from the value it could be determined whether its a mac or an IP (by the existence of a `:` in the value ?)

Wdyt ?